example-java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
Should remove these files, this example dir was left in my local copy of the repo but has been removed from upstream.

Reply:
So wait, did you remove these? Or are you saying that should be in a different PR?

Reply:
About to delete them and push again.